src/ens.js

```diff
@@ -556,7 +498,12 @@ export class ENS {
     let namehash = getNamehash(reverseNode)
     return Resolver.setName(namehash, name)
   }
-
+  async wildcardResolverDomain(name){
```

Comment:
This definitely needs a different name - something like 'hasResolver'?

Reply:
This should at least be 'hasWildcardResolver' or somesuch; the current name implies it returns a 'domain', not a bool.